playwright/e2e/clusters/cluster-list-validation.spec.ts

```
@@ -0,0 +1,298 @@
+      const helpIcon = clusterListPage.viewOnlyMyClusterHelp();
+      await expect(helpIcon).toBeVisible();
+    });
+
+    test('should display cluster table with correct headers', async ({ clusterListPage }) => {
+      // Verify all table column headers
+      const expectedHeaders = ['Name', 'Status', 'Type', 'Created', 'Version', 'Provider (Region)'];
+
+      for (const header of expectedHeaders) {
+        const headerCell = clusterListPage.tableHeader(header);
+        await expect(headerCell).toBeVisible();
+      }
```

ESLint

[error] iterators/generators require regenerator-runtime, which is too heavyweight for this guide to allow them. Separately, loops should be avoided in favor of array iterations.

(no-restricted-syntax)